docs/api/actions.md

```diff
@@ -159,6 +159,26 @@ is rejected or cancelled with the same payload.
 When all steps of the generator complete, the payload of the parent
 action will be the resolved payload of the final action.
 
+#### Yielding Actions in Parallel
+
+By yielding an array of actions, you can wait for multiple actions to
+complete before continuing:
+
+```javascript
+function getUser (id) {
+  return fetch(`/users/${id}`)
```

Comment:
I just pushed a thing to master. `fetch` has a couple of problems, like it doesn't reject the promise it returns if it receives a 400. I've updated my examples to read:
```
import axios from 'axios'

function getUser (id) {
  // axios returns a promise
  return axios(`/users/${id}`)
}
```